2023-03-09 16:06:52,514 INFO    Thread-17 :42672 [wandb_setup.py:_flush():68] Configure stats pid to 42672
2023-03-09 16:06:52,514 INFO    Thread-17 :42672 [wandb_setup.py:_flush():68] Loading settings from C:\Users\Shivam Maurya\.config\wandb\settings
2023-03-09 16:06:52,514 INFO    Thread-17 :42672 [wandb_setup.py:_flush():68] Loading settings from C:\Users\Shivam Maurya\OneDrive\Desktop\IIT MADRAS\Deep Learning\assignment 1\CS6910-Deep_Learning_Assignment-1\wandb\settings
2023-03-09 16:06:52,514 INFO    Thread-17 :42672 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True', 'project': 'CS6910_Assignment_1', 'entity': 'cs22m083', 'root_dir': 'C:\\Users\\Shivam Maurya\\OneDrive\\Desktop\\IIT MADRAS\\Deep Learning\\assignment 1\\CS6910-Deep_Learning_Assignment-1', 'sweep_id': 'd5gozo0e', 'run_id': 'o2qujscg', 'sweep_param_path': 'C:\\Users\\Shivam Maurya\\OneDrive\\Desktop\\IIT MADRAS\\Deep Learning\\assignment 1\\CS6910-Deep_Learning_Assignment-1\\wandb\\sweep-d5gozo0e\\config-o2qujscg.yaml'}
2023-03-09 16:06:52,514 INFO    Thread-17 :42672 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-03-09 16:06:52,514 INFO    Thread-17 :42672 [wandb_init.py:_log_setup():492] Logging user logs to C:\Users\Shivam Maurya\OneDrive\Desktop\IIT MADRAS\Deep Learning\assignment 1\CS6910-Deep_Learning_Assignment-1\wandb\run-20230309_160652-o2qujscg\logs\debug.log
2023-03-09 16:06:52,514 INFO    Thread-17 :42672 [wandb_init.py:_log_setup():493] Logging internal logs to C:\Users\Shivam Maurya\OneDrive\Desktop\IIT MADRAS\Deep Learning\assignment 1\CS6910-Deep_Learning_Assignment-1\wandb\run-20230309_160652-o2qujscg\logs\debug-internal.log
2023-03-09 16:06:52,514 INFO    Thread-17 :42672 [wandb_init.py:init():532] calling init triggers
2023-03-09 16:06:52,514 INFO    Thread-17 :42672 [wandb_init.py:init():538] wandb.init called with sweep_config: {'activation': 'relu', 'batch_size': 16, 'epochs': 10, 'gamma': 0.001, 'hidden_layers': [32, 64, 128], 'learning_rate': 0.0001, 'optimizer': 'adam', 'weight_decay': 0.5}
config: {}
2023-03-09 16:06:52,514 INFO    Thread-17 :42672 [wandb_init.py:init():554] re-initializing run, found existing run on stack: o2qujscg
2023-03-09 16:06:52,514 INFO    Thread-17 :42672 [wandb_run.py:_finish():1854] finishing run cs22m083/CS6910_Assignment_1/o2qujscg
2023-03-09 16:06:52,514 INFO    Thread-17 :42672 [jupyter.py:save_history():449] not saving jupyter history
2023-03-09 16:06:52,514 INFO    Thread-17 :42672 [jupyter.py:save_ipynb():379] not saving jupyter notebook
2023-03-09 16:06:52,514 INFO    Thread-17 :42672 [wandb_init.py:_jupyter_teardown():420] cleaning up jupyter logic
2023-03-09 16:06:52,514 INFO    Thread-17 :42672 [wandb_run.py:_atexit_cleanup():2083] got exitcode: 0
2023-03-09 16:06:52,514 INFO    Thread-17 :42672 [wandb_run.py:_restore():2066] restore
2023-03-09 16:06:52,514 INFO    Thread-17 :42672 [wandb_run.py:_restore():2072] restore done
2023-03-09 16:06:57,473 INFO    Thread-17 :42672 [wandb_run.py:_footer_history_summary_info():3442] rendering history
2023-03-09 16:06:57,473 INFO    Thread-17 :42672 [wandb_run.py:_footer_history_summary_info():3474] rendering summary
2023-03-09 16:06:57,473 INFO    Thread-17 :42672 [wandb_run.py:_footer_sync_info():3398] logging synced files
2023-03-09 16:06:57,473 INFO    Thread-17 :42672 [wandb_init.py:init():588] starting backend
2023-03-09 16:06:57,473 INFO    Thread-17 :42672 [wandb_init.py:init():592] setting up manager
2023-03-09 16:06:57,490 INFO    Thread-17 :42672 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn, using: spawn
2023-03-09 16:06:57,493 INFO    Thread-17 :42672 [wandb_init.py:init():599] backend started and connected
2023-03-09 16:06:57,506 INFO    Thread-17 :42672 [wandb_run.py:_config_callback():1250] config_cb None None {'activation': 'relu', 'batch_size': 16, 'epochs': 10, 'gamma': 0.001, 'hidden_layers': [32, 64, 128], 'learning_rate': 0.0001, 'optimizer': 'adam', 'weight_decay': 0.5}
2023-03-09 16:06:57,509 INFO    Thread-17 :42672 [wandb_run.py:_label_probe_notebook():1203] probe notebook
2023-03-09 16:06:57,509 INFO    Thread-17 :42672 [wandb_run.py:_label_probe_notebook():1213] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-03-09 16:06:57,509 INFO    Thread-17 :42672 [wandb_init.py:init():687] updated telemetry
2023-03-09 16:06:57,571 INFO    Thread-17 :42672 [wandb_init.py:init():727] communicating run to backend with 60.0 second timeout
2023-03-09 16:06:58,218 INFO    Thread-17 :42672 [wandb_run.py:_on_init():2134] communicating current version
2023-03-09 16:06:58,260 INFO    Thread-17 :42672 [wandb_run.py:_on_init():2143] got version response upgrade_message: "wandb version 0.13.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-03-09 16:06:58,260 INFO    Thread-17 :42672 [wandb_init.py:init():775] starting run threads in backend
2023-03-09 16:07:07,119 INFO    Thread-17 :42672 [wandb_run.py:_console_start():2114] atexit reg
2023-03-09 16:07:07,119 INFO    Thread-17 :42672 [wandb_run.py:_redirect():1969] redirect: SettingsConsole.WRAP_RAW
2023-03-09 16:07:07,119 INFO    Thread-17 :42672 [wandb_run.py:_redirect():2034] Wrapping output streams.
2023-03-09 16:07:07,119 INFO    Thread-17 :42672 [wandb_run.py:_redirect():2059] Redirects installed.
2023-03-09 16:07:07,119 INFO    Thread-17 :42672 [wandb_init.py:init():817] run started, returning control to user process
2023-03-09 16:07:07,126 INFO    Thread-17 :42672 [wandb_run.py:_finish():1854] finishing run cs22m083/CS6910_Assignment_1/o2qujscg
2023-03-09 16:07:07,126 ERROR   MainThread:42672 [pyagent.py:_run_jobs_from_queue():226] Run o2qujscg errored: NameError("name 'x_val' is not defined")
2023-03-09 16:22:17,426 ERROR   Thread-16 :42672 [internal_api.py:execute():245] 502 response executing GraphQL.
2023-03-09 16:22:17,426 ERROR   Thread-16 :42672 [internal_api.py:execute():246] 
<html><head>
<meta http-equiv="content-type" content="text/html;charset=utf-8">
<title>502 Server Error</title>
</head>
<body text=#000000 bgcolor=#ffffff>
<h1>Error: Server Error</h1>
<h2>The server encountered a temporary error and could not complete your request.<p>Please try again in 30 seconds.</h2>
<h2></h2>
</body></html>

2023-03-09 16:25:03,736 ERROR   Thread-16 :42672 [internal_api.py:execute():245] 502 response executing GraphQL.
2023-03-09 16:25:03,736 ERROR   Thread-16 :42672 [internal_api.py:execute():246] 
<html><head>
<meta http-equiv="content-type" content="text/html;charset=utf-8">
<title>502 Server Error</title>
</head>
<body text=#000000 bgcolor=#ffffff>
<h1>Error: Server Error</h1>
<h2>The server encountered a temporary error and could not complete your request.<p>Please try again in 30 seconds.</h2>
<h2></h2>
</body></html>

2023-03-09 19:54:26,999 ERROR   Thread-16 :42672 [internal_api.py:execute():245] 502 response executing GraphQL.
2023-03-09 19:54:26,999 ERROR   Thread-16 :42672 [internal_api.py:execute():246] 
<html><head>
<meta http-equiv="content-type" content="text/html;charset=utf-8">
<title>502 Server Error</title>
</head>
<body text=#000000 bgcolor=#ffffff>
<h1>Error: Server Error</h1>
<h2>The server encountered a temporary error and could not complete your request.<p>Please try again in 30 seconds.</h2>
<h2></h2>
</body></html>

2023-03-09 20:46:19,844 ERROR   Thread-16 :42672 [internal_api.py:execute():245] 502 response executing GraphQL.
2023-03-09 20:46:19,844 ERROR   Thread-16 :42672 [internal_api.py:execute():246] 
<html><head>
<meta http-equiv="content-type" content="text/html;charset=utf-8">
<title>502 Server Error</title>
</head>
<body text=#000000 bgcolor=#ffffff>
<h1>Error: Server Error</h1>
<h2>The server encountered a temporary error and could not complete your request.<p>Please try again in 30 seconds.</h2>
<h2></h2>
</body></html>

2023-03-09 20:53:48,776 ERROR   Thread-16 :42672 [internal_api.py:execute():245] 502 response executing GraphQL.
2023-03-09 20:53:48,776 ERROR   Thread-16 :42672 [internal_api.py:execute():246] 
<html><head>
<meta http-equiv="content-type" content="text/html;charset=utf-8">
<title>502 Server Error</title>
</head>
<body text=#000000 bgcolor=#ffffff>
<h1>Error: Server Error</h1>
<h2>The server encountered a temporary error and could not complete your request.<p>Please try again in 30 seconds.</h2>
<h2></h2>
</body></html>

2023-03-09 22:22:40,456 ERROR   Thread-16 :42672 [internal_api.py:execute():245] 404 response executing GraphQL.
2023-03-09 22:22:40,459 ERROR   Thread-16 :42672 [internal_api.py:execute():246] {"errors":[{"message":"could not find agent 79kgnfuv during agentHeartbeat","path":["agentHeartbeat"]}],"data":{"agentHeartbeat":null}}
